src/components/Gallery.js

```diff
@@ -1,9 +1,12 @@
 import { h } from 'preact'
-import { useState } from 'preact/hooks'
+import { useState, useRef, useLayoutEffect } from 'preact/hooks'
 import { useI18n, useSoftkey, usePopup } from 'hooks'
 
+const MAX_DESCRIPTION_HEIGHT = 44
```

Comment:
How did you arrive at that number? It should be exactly 3 x line-height

Reply:
I didn't set the line-height for the text here, and 44 is the best match with `...`

or I can set the `line-height: 15px` and `max-height: 45px`